README.md

```diff
@@ -9,10 +9,27 @@ Application Inspector is different from traditional static analysis tools in tha
 
 The tool supports scanning various programming languages including C, C++, C#, Java, JavaScript, HTML, Python, Objective-C, Go, Ruby, PowerShell and [more](https://github.com/microsoft/ApplicationInspector/wiki/3.4-Applies_to-(languages)) and can scan projects with mixed language files.  It supports generating results in HTML, JSON and text output formats with the **default being an HTML report** similar to the one shown here.
 
-![AppInspector-Features](https://user-images.githubusercontent.com/47648296/72893326-9c82c700-3ccd-11ea-8944-9831ea17f3e0.png)
+![appinspector-Features](https://user-images.githubusercontent.com/47648296/72893326-9c82c700-3ccd-11ea-8944-9831ea17f3e0.png)
 
 Be sure to see our complete project wiki page https://Github.com/Microsoft/ApplicationInspector/wiki for additional information and help.
 
+# Quick Start
+## Obtain Application Inspector
+### .NET Tool (recommended)
+- Download and install the .NET Core 5.0 [SDK](https://dotnet.microsoft.com/download/dotnet-core/5.0)
+- Run `dotnet tool install --global Microsoft.CST.ApplicationInspector.CLI` 
+
+See more in the [wiki](https://github.com/microsoft/ApplicationInspector/wiki/7.-NuGet-Support)
+
+### Platform Dependent Binary
+- Download Application Inspector by selecting the pre-built package for the operating system of choice shown under the Assets section
+of the [Releases](https://github.com/microsoft/ApplicationInspector/releases).
+
+## Run Application Inspector
+
+- Nuget Tool: `appinspector analyze -s path/to/src`.
+- Platform Specific: `applicationinspector.cli.exe analyze -s path/to/src`
```

Comment:
These are supposed to be different, right? appinspector vs. applicationinspector.cli.exe?

Reply:
Yes.

The nuget tool command is `appinspector`

The raw CLI binary is named `applicationinspector.cli` with `.exe` on windows.